earth_enterprise/BUILD.md

```diff
@@ -61,15 +69,16 @@ platforms on how to setup the dependencies, tools, and compilers.
         the configuration to run again, otherwise the scons build may complain
         about missing libraries
 
-1. Build Earth Enterprise Fusion and Server:
+2. Build Earth Enterprise Fusion and Server:
```

Comment:
I'm not positive this is needed but it also doesn't appear to hurt (see line 45 above and view it in the preview).

Also should line 24 be a continuation of the numbered list or start over at 1?  IMO it should be a continuation but no idea how to make the markdown interpret it as such.

Reply:
Yes, it does look like step 1 occurs three times in the above text.